Unable to build and run `rusk` node binary

**Describe the bug**
Unable to build and run `rusk` node.
It's worth to mention that `make build` works perfectly, and that's maybe why we didn't catch the issue: if we start have more logic on the binary as well, we should probably build not just the library but the binary too in our CI.

**To Reproduce**
Execute `make run`

**Expected behaviour**
`make run` should build the node and run the node.

**Logs/Screenshot**
```
cargo run --release --bin rusk
  Downloaded fake v2.9.2
  Downloaded 1 crate (104.7 KB) in 0.45s
   Compiling dusk-poseidon v0.31.0
   Compiling dlmalloc v0.2.5
   Compiling serde_spanned v0.6.5
   Compiling toml_datetime v0.6.5
   Compiling rusk v0.7.0 (/Users/zer0/dusk/rusk/rusk)
   Compiling toml_edit v0.19.15
   Compiling dusk-pki v0.13.0
   Compiling poseidon-merkle v0.3.0
   Compiling dusk-schnorr v0.14.0
   Compiling phoenix-core v0.21.0
   Compiling stake-contract-types v0.0.1-rc.2
   Compiling stake-contract-types v0.0.1-rc.2 (/Users/zer0/dusk/rusk/contracts/stake-types)
   Compiling piecrust-uplink v0.9.0
   Compiling rusk-abi v0.12.0-rc.0
error[E0152]: found duplicate lang item `panic_impl`
  --> /Users/zer0/.cargo/registry/src/index.crates.io-6f17d22bba15001f/piecrust-uplink-0.9.0/src/abi/handlers.rs:15:1
   |
15 | unsafe fn handle_panic(info: &PanicInfo) -> ! {
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: the lang item is first defined in crate `std` (which `rkyv` depends on)
   = note: first definition in `std` loaded from /Users/zer0/.rustup/toolchains/nightly-2023-05-22-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/libstd-8c15124082ee9a2d.rlib
   = note: second definition in the local crate (`piecrust_uplink`)

error[E0152]: found duplicate lang item `eh_personality`
  --> /Users/zer0/.cargo/registry/src/index.crates.io-6f17d22bba15001f/piecrust-uplink-0.9.0/src/abi/handlers.rs:32:1
   |
32 | extern "C" fn eh_personality() {}
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: the lang item is first defined in crate `std` (which `rkyv` depends on)
   = note: first definition in `std` loaded from /Users/zer0/.rustup/toolchains/nightly-2023-05-22-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/libstd-8c15124082ee9a2d.rlib
   = note: second definition in the local crate (`piecrust_uplink`)

For more information about this error, try `rustc --explain E0152`.
error: could not compile `piecrust-uplink` (lib) due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
make: *** [run] Error 101
```

**Platform**
Fill as appropriate
 - Architecture: Apple Silicon
 - OS: macOS Sonoma

--- a/Makefile
+++ b/Makefile
@@ -52,7 +52,7 @@ clippy: ## Run clippy
 	$(MAKE) -C ./rusk/ $@
 
 run: keys state web-wallet ## Run the server
-	cargo run --release --bin rusk
+	$(MAKE) -C ./rusk/ $@
 
 rusk: keys state web-wallet ## Build rusk binary
 	$(MAKE) -C ./rusk build

--- a/rusk/Makefile
+++ b/rusk/Makefile
@@ -20,10 +20,13 @@ clippy: ## Run clippy
 build: ## Build rusk binary
 	@cargo build --release
 
+run:
+	@cargo r --release --bin rusk
+
 recovery-keys: ## Build circuit keys
 	@cargo r --no-default-features --features recovery-keys --release -- recovery-keys
 
 recovery-state: ## Build network state
 	@cargo r --no-default-features --features recovery-state --release -- recovery-state --init $(INITFILE)
 
-.PHONY: test help clippy build recovery-keys recovery-state
+.PHONY: test help clippy build recovery-keys recovery-state rusk

